distro/wildfly/modules/src/main/modules/org/camunda/bpm/dmn/camunda-engine-feel-scala/main/module.xml

```diff
@@ -10,8 +10,8 @@
     <module name="org.camunda.commons.camunda-commons-typed-values" />
     <module name="org.camunda.commons.camunda-commons-logging" />
 
-    <module name="org.camunda.bpm.camunda-engine-plugin-spin" optional="true" />
+    <module name="org.camunda.bpm.camunda-engine-plugin-spin" services="import" export="true"/>
```

Comment:
💭 So far the Spin plugin was optional for users, I think it should remind like this and users that don't use the plugin, should not be able to use the new feature for spin values.

Reply:
TODO: put findings for future reference.